packages/react/src/components/pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { Pagination, createPaginationItemList } from './Pagination';
+
+describe('createPaginationItemList', () => {
```

Comment:
Would it possible to test these using the public component API?

Reply:
It should be possible:

- [ ] Test pagination item list using the public component API